exodus/exodus/locale/fr/LC_MESSAGES/django.po

```diff
@@ -154,44 +155,47 @@ msgstr ""
 "à propos de vous et de vos usages.</b> Ainsi, les rapports d’εxodus vous "
 "révèlent les ingrédients du gâteau."
 
-#: templates/home.html:48
-msgid "εxodus does not decompile applications, its analysis technic is legal."
+#: exodus/templates/home.html:48
+#, fuzzy
+#| msgid ""
+#| "εxodus does not decompile applications, its analysis technic is legal."
```

Comment:
We can remove these 2 lines

Reply:
No. It's the output of the Django `makemessages` command.

Reply:
Actually we can and should.
Fuzzy just means that the translation changed and django is unsure of the right translation so it adds this comment.

If the translation is correct the right practice is to remove "fuzzy" comment.